scripts/ci/iptables-restore

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
```

Comment:
add a header to explain why in oneline, like the k8s patch
add this PR as reference

Reply:
ok